eureka-webapp/src/main/resources/Messages.properties

```diff
@@ -49,7 +49,7 @@ phenotypetype.FREQUENCY.description=For defining a disease, finding or patient c
 phenotypetype.VALUE_THRESHOLD.displayName=Value Threshold
 phenotypetype.VALUE_THRESHOLD.description=For defining clinically significant thresholds on the value of an observation.
 passwordChange.error.internalServerError=Error while changing password.
-deletePhenotype.error.internalServerError=Error trying to delete phenotype. The phenotype is in use. There is a problem with Eureka!.
+deletePhenotype.error.internalServerError=Error trying to delete phenotype. The phenotype is in use. There is a problem with Eureka!
```

Comment:
If the error is really caused by phenotype in use, the error msg looks like this:
<img width="620" alt="screen shot 2016-05-12 at 2 29 06 pm" src="https://cloud.githubusercontent.com/assets/5263886/15225806/1f17f6e8-184e-11e6-9867-f9d27b41aaba.png">